### ISSUE TITLE
Consider including py.typed file in distribution

### ISSUE THREAD

**Issue by jheiss** [Superbird11/ranges#20]
Since it looks like you've added type annotations consider including a py.typed file in the distribution so that users can do type checking against your library. E.g. right now I'm seeing

```$ mypy mycode.py                      
mycode.py:44: error: Skipping analyzing "ranges": module is installed, but missing library stubs or py.typed marker
mycode.py:44: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

**Comment by jheiss** [comment]
You added py.typed on Oct 9, and released 1.2.1 on the same day so I'm guessing it should have picked up the change. But I don't see py.typed in my installed package.

```
$ pip list | grep python-ranges
python-ranges                    1.2.1
$ ls venv/lib/python3.10/site-packages/ranges
Range.py	RangeDict.py	RangeSet.py	__init__.py	__pycache__	_helper.py
```

**Comment by jheiss** [comment]
Possibly a typo in setup.py?

```
"ranges": ["py.types"]
```
